c/meterpreter/source/common/common_logging.c

```diff
@@ -0,0 +1,42 @@
+#include "common.h"
+HANDLE lock = NULL;
+HANDLE hFile = NULL;
+
+HANDLE initLogging(wchar_t* filePath) {
+	hFile = CreateFileW(filePath,                // name of the write
+		GENERIC_WRITE,          // open for writing
+		FILE_SHARE_DELETE | FILE_SHARE_READ | FILE_SHARE_WRITE,                      // do share (7)
+		NULL,                   // default security
+		CREATE_ALWAYS,             // create new file always
+		FILE_ATTRIBUTE_NORMAL,  // normal file
+		NULL);                  // no attr. template
+	lock = CreateMutex(NULL, FALSE, NULL);
+
+	if (hFile == NULL) {
+        dprintf("[LOGGING] Logging to file failed to initialize")
```

Comment:
```suggestion
        dprintf("[LOGGING] Logging to file failed to initialize");
```

Reply:
ah thanks :sweat_smile: I had this fixed just didn't get commited